rubygem-infoblox/rubygem-infoblox.spec

```diff
@@ -0,0 +1,84 @@
+# Generated from infoblox-1.0.0.gem by gem2rpm -*- rpm-spec -*-
+%global gem_name infoblox
+
+Name: rubygem-%{gem_name}
+Version: 1.0.0
+Release: 1%{?dist}
+Summary: A Ruby wrapper to the Infoblox WAPI
+Group: Development/Languages
+License: MIT
+URL: https://github.com/govdelivery/infoblox
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+%if 0%{?rhel} == 6
+Requires: ruby(abi)
+%else
+Requires: ruby(release)
+%endif
+Requires: ruby
+Requires: ruby(rubygems)
+Requires: rubygem(faraday)
+Requires: rubygem(faraday_middleware)
```

Comment:
faraday is required by packages in the core repo, while faraday_middleware is only required by plugins, hence the split.

faraday has a dependency on multipart-post, which also isn't satisfied. Please check the package is installable.